assets/js/blocks/cart/inner-blocks/cart-order-summary-taxes/edit.tsx

```diff
@@ -63,6 +64,7 @@ export const Edit = ( {
 							/>
 						</PanelBody>
 					) }
+				<CartSidebarCompatibilityNotice />
```

Comment:
Same here! We should place the alert under the `Name & description` panel before the Block's settings panels (i.e., Let's move it to `line 39`).